crawler.js

```diff
@@ -654,14 +683,20 @@ class Crawler {
       if (!fs.existsSync(this.pagesDir)) {
         fs.mkdirSync(this.pagesDir);
         const header = {"format": "json-pages-1.0", "id": "pages", "title": "All Pages"};
+        header["hasText"] = this.params.text;
+        header["hasReaderView"] = this.params.readerView;
```

Comment:
Instead of adding "hasReaderView" how about using "textSource". That way we can reference using readability or the browser dom or 'boilerpipe'. the boilerpipe library lives in py-wacz and we have the dom extraction method used in browsertrix-crawler and in archiveweb.page. This way we can be specific about what method was used for text extraction. 

So the current default would be browser-dom and when reader view is set it would instead be 'readability'

Reply:
Sure, I can change this. Just to confirm, you mean?
```javascript
        header["textSource"] = (this.params.readerView ? "readability" : "browser-dom");
```

Also: by now, the `article` object is added to a page in case there is the reader view is available, see [isProbablyReaderable](https://github.com/mozilla/readability#isprobablyreaderabledocument-options). If the intention is to replace title and text by the readerable values, shouldn't the property set per page?